server/plugin.go

```diff
@@ -142,6 +152,30 @@ func signClaims(secret string, claims *Claims) (string, error) {
 	return string(token.Raw()), nil
 }
 
+func (p *Plugin) trackMeeting(args *model.CommandArgs) {
+	// disables tracking if the user is not using the default jitsi url
+	isDefaultURL := p.isDefaultJitsiURL()
+	if !isDefaultURL {
+		return
+	}
```

Comment:
Is tracking which users use the default URL the whole point of the ticket?

Reply:
@hanzei  yup

Reply:
@hanzei  I might have misunderstood your comment here. Earlier I was tracking the link  but changed based on the comment below. Please advise
https://github.com/mattermost/mattermost-plugin-jitsi/pull/194#discussion_r655634421